xdsl/printer.py

```diff
@@ -184,6 +184,21 @@ def print_dictionary(
             self.print("=")
             print_value(value)
 
+    def print_attribute_dictionary(
```

Comment:
I guess `V` should be `Attribute` here?

Reply:
Yup, and K just str, right?

Reply:
Oh, so we can get rid of the Callable arguments as a whole too!

Reply:
Yes! K would be `str`. I would just remove all parameters to be honest!
Also, I would argue that we don't actually need the normal `print_dictionary`, but that's for another time!

Reply:
Sure!